### ISSUE TITLE
False positive [POP-401] when secret is both mounted and used in valueFrom

### ISSUE THREAD

**Issue by robwruck** [derailed/popeye#161]
<img src="https://raw.githubusercontent.com/derailed/popeye/master/assets/popeye_bug.png" align="right" width="100" height="auto"/>

<br/>
<br/>
<br/>

**Describe the bug**
When some entries of a secret are mounted as volumes *and* some are used in `valueFrom`, those used in `valueFrom` are reported as `[POP-401] Key "name" used? Unable to locate key reference.`

**To Reproduce**
- Deploy this:
```
apiVersion: v1
kind: Secret
type: Opaque
metadata:
  name: popeye
data:
  file: ZmlsZQ==
  text: dGV4dA==

---

apiVersion: apps/v1
kind: Deployment
metadata:
  name: popeye
spec:
  replicas: 1
  selector:
    matchLabels:
      app: popeye
  template:
    metadata:
      labels:
        app: popeye
    spec:
      volumes:
        - name: secret
          secret:
            secretName: popeye
            items:
              - key: file
                path: file
      containers:
      - name: nginx
        image: nginx:1.17.5-alpine
        imagePullPolicy: Always
        volumeMounts:
        - name: secret
          mountPath: "/etc/nginx/conf.d"
          readOnly: true
        ports:
        - name: http
          containerPort: 80
        env:
        - name: SOME_TEXT
          valueFrom:
            secretKeyRef:
              name: popeye
              key: text
        resources:
          limits:
            cpu: 100m
            memory: 64Mi
          requests:
            cpu: 10m
            memory: 24Mi
```
- Run popeye

**Expected behavior**
No POP-401 is reported.
This is the case when both `volume` and `volumeMounts` are removed and the second secret entry is also used in a `valueFrom`.

**Screenshots**
n/a

**Versions (please complete the following information):**
 - Popeye 0.9.0
 - K8s 1.14

**Additional context**
n/a

**Comment by derailed** [comment]
@robwruck Great catch! Thank you. Will fix in the next drop.

**Comment by derailed** [comment]
@robwruck Fixed! v0.9.1
